data_gateway/cli.py

```diff
@@ -45,11 +45,15 @@ def gateway_cli(logger_uri, log_level):
     from octue.log_handlers import apply_log_handler, get_remote_handler
 
     # Apply log handler locally.
-    apply_log_handler(log_level=log_level.upper())
+    apply_log_handler(log_level=log_level.upper(), include_thread_name=True)
 
     # Stream logs to remote handler if required.
     if logger_uri is not None:
-        apply_log_handler(handler=get_remote_handler(logger_uri=logger_uri), log_level=log_level.upper())
+        apply_log_handler(
+            handler=get_remote_handler(logger_uri=logger_uri),
+            log_level=log_level.upper(),
+            include_thread_name=True,
+        )
```

Comment:
Why are we applying the log handler twice? Shouldn't this be an if-else?

Reply:
If no logger URI is provided, just a local log handler is applied; if a URI is provided, then the local handler and the remote handler are applied so logs can be seen locally and remotely. I was wondering actually whether remote logging is something we need to provide in `data-gateway`?

Reply:
Oh! Of course, got it.

Remote logging: lets not do it until it's deemed that we need it ;)

Reply:
Ok haha I might remove it then